src/VisualStudio/Core/Def/Packaging/IPackageSearchIOService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.LanguageServices.Packaging
+{
+    /// <summary>
+    /// Used so we can mock out how the search service does IO for testing purposes.
+    /// </summary>
+    internal interface IPackageSearchIOService
+    {
```

Comment:
This is an interface purely to allow mocking of the io operations that the PackageSearchService needs (for testing purposes).  Hence PackageSearchIOService.
  We could consider using async IO here.  Though i couldn't find async versions of most of these.